sunpy/util/metadata.py

```diff
@@ -34,11 +33,14 @@ def __init__(self, *args):
             args = list(args)
             adict = args[0]
             if isinstance(adict, list) or isinstance(adict, tuple):
-                tags = OrderedDict((k.lower(), v) for k, v in adict)
+                items = adict
```

Comment:
This isn't related to the this PR but if its a tuple or a list, how does it have a "k,v" for the code below?

Reply:
I think because it's meant to be a list of lists, where each sub-list is a 2-length key-item pair, e.g. `[['key1', 'item1'], ['key2', 'item2']]`